web/src/people.ts

```diff
@@ -995,6 +995,14 @@ export function get_realm_users(): User[] {
     return [...active_user_dict.values()];
 }
 
+export function get_realm_users_and_welcome_bot(): (User | CrossRealmBot)[] {
```

Comment:
We should change `User | CrossRealmBot` once https://github.com/zulip/zulip/pull/30755 gets merged. I realised `User | CrossRealmBot` does not make sense :)

Reply:
Yes this is temporary. I initially used `User` only but thought we should maintain types atleast until they exist.

Reply:
Dropped `CrossRealmBot` since 175dee80dd800a759fb4c63e6b78cf809ffcb89f got merged.